app/controllers/messages_controller.rb

```diff
@@ -20,6 +20,6 @@ def conversation
   end
 
   def message_params
-    params.require(:message).permit(:text)
+    jsonapi_deserialize(params, only: [:text])
```

Comment:
no need for []

Reply:
I think we do need, according to the docs https://github.com/stas/jsonapi.rb#deserialization